monitorcontroller/monitorcontroller.go

```diff
@@ -39,27 +42,23 @@ import (
 type MonitoringSender interface {
 	SubscribeForConnectionEvents(consumer amqphandler.ConnectionEventsConsumer) error
 	UnsubscribeFromConnectionEvents(consumer amqphandler.ConnectionEventsConsumer) error
-	SendMonitoringData(monitoringData cloudprotocol.Monitoring) error
+	SendNodeMonitoring(monitoringData cloudprotocol.Monitoring) error
 }
 
 // MonitorController instance.
 type MonitorController struct {
 	sync.Mutex
-	monitoringQueue     []cloudprotocol.NodeMonitoringData
-	monitoringQueueSize int
-	monitoringSender    MonitoringSender
-	cancelFunction      context.CancelFunc
-	isConnected         bool
-}
+	offlineMessages []cloudprotocol.Monitoring
 
-/***********************************************************************************************************************
- * Vars
- **********************************************************************************************************************/
+	sendMessageEvent   chan bool
```

Comment:
Use empty struct type for notifications without data `struct{}`

Reply:
done